.azure-pipelines/apiview.yml

```diff
@@ -183,18 +183,13 @@ stages:
               pwsh: true
             displayName: 'Start Azurite'
 
-          - task: DotNetCoreInstaller@2
-            displayName: 'Use .NET Core sdk 3.1.x'
+          - task: UseDotNet@2
+            displayName: 'Install .NET'
             inputs:
-              version: '3.1.x'
-          
-          - task: DotNetCoreInstaller@2
-            displayName: 'Use .NET Core sdk 5.x'
-            inputs:
-              version: '5.x'
+              version: '$(DotNetRuntimeVersion)'
 
-          - task: DotNetCoreInstaller@2
-            displayName: 'Use .NET Core $(DotNetRuntimeVersion)'
+          - task: UseDotNet@2
+            displayName: 'Install .NET RunTime'
             inputs:
               packageType: runtime
```

Comment:
Do we need both the SDK and runtime installed independently? I wouldn't expect we would.